mobify/bellows
# Iframe test

Updating test infrastructure to use iframe method.

Status: **Ready to merge**

Reviewers: @Helen-Mobify 

## Changes
- updated all tests to use new iframe sandbox method for testing
- added requirejs-glob for test files

## How to test-drive this PR
- checkout this branch
- `grunt test` and also `grunt test:browser`
- make sure stuff works
